server/controllers/personal/personalController.test.ts

```diff
@@ -291,7 +291,7 @@ describe('PersonalController', () => {
             user: prisonUserMock,
             prisonerNumber: 'ABC123',
             correlationId: request.id,
-            action: PostAction.EditPhysicalCharacteristics,
+            action: PostAction.EditHeight,
```

Comment:
Since we're no longer editing physical characteristics in the Prison Person API, and we edit each field individually I thought it made sense to actually have a separate action for each edit

Reply:
OR we could make these actions all more generic, maybe EDIT_PROFILE perhaps? Because we record the fieldName in the details 🤷